myapp/spec/system/tasks_spec.rb

```diff
@@ -4,7 +4,24 @@
 
 RSpec.describe "Tasks", type: :system do
   before do
-    @task = Task.create!(title: "test task title", description: "test task description")
```

Comment:
in rspec instead of using @ variable, it's better to use let and let! to declare the variables.